config/default.json

```diff
@@ -14,7 +14,20 @@
   "validations": {
     "email": "^\\w+([\\.-]?\\w+)*@\\w+([\\.-]?\\w+)*(\\.\\w{2,3})+$"
   },
+  "auth": {
+    "excludeAuthPaths": [
+      {
+        "method": "POST",
+        "endpoint": "/api/identity"
+      }
+    ]
+  },
+  "uniqueness": {
+    "devices": ["sms", "email", "apn", "gcm", "phone"],
+    "channels": true
+  },
   "test": {
-    "fixtures_folder": "../fixtures/"
+    "fixtures_folder": "/fixtures/",
+    "resources_folder": "/cucumber/test_files/"
```

Comment:
@guumaster The reason to put that config there is that fixtures could be loaded in a non-testing enviroment (that is, one could load the fixtures to the local database to double-check the objects manipulation).

Of course, if we disregard this single case then we could move the test configuration to the config.test file.